axelor-production/src/main/resources/views/ManufOrder.xml

```diff
@@ -3,9 +3,13 @@
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xsi:schemaLocation="http://axelor.com/xml/ns/object-views http://axelor.com/xml/ns/object-views/object-views_5.2.xsd">
 
-  <grid name="manuf-order-grid" title="Manufacturing orders" model="com.axelor.apps.production.db.ManufOrder" orderBy="-manufOrderSeq">
```

Comment:
I cannot accept 700 lines of re-indentation. 
There are probably things you actually changed in there, but I have no idea why and it also causes merge conflicts afterwards. 
If the code needs to be adapted to a specific code standard, it should be done in a separate commit.